docs/notebooks/des-gaia.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>" due to margin caches" -&gt; quite unclear, elabore more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>